gitlens/gitlens-release-notes-current.md

```diff
@@ -16,6 +16,88 @@ Features marked with a ☁️ require a GitKraken Account, with access level bas
 
 ---
 
+
+<a id="v14-6"></a>
+
+## Version 14.6
+
+#### Wednesday, December 13, 2023
+
+GitLens 14.6 marks our last release of 2023 and with it some great enhancements to Cloud Patches, Focus View and much more!
+
+### Cloud Patch Sharing Options ☁️
+
+Users can now specify who can access shared Cloud Patches, choosing between anyone with a link or only members of your GitKraken organization with the link. 
```

Comment:
Should we mention that users with multiple organizations can select one in the GitKraken account view to share private cloud patches with that organization? Would need some word-smithing.